design/secrets.md

```diff
@@ -84,6 +147,22 @@ Currently, GCP is the only provider that relies on the existing environment vari
 For GCP, the environment variable will be overwritten with the path of the serialized secret.
 
 
+## Split credentials between VolumeSnapshotter and ObjectStore plugins
+
+One of the use cases we wish to satisfy is the ability to specify a different object store than the cloud provider offers,
+for example, using a Minio S3 object store from within AWS.  Currently the VolumeSnapshotter and the ObjectStore plugin
+share the cloud credentials.  The VolumeSnapshotter cannot use the Minio credentials to take snapshots.  We need to split
+the VolumeSnapshotter credentials from the ObjectStore credentials.  We will do this by always using the default cloud
+credentials with the VolumeSnapshotter plugins and not passing the BSL credentials in the current implementation.
```

Comment:
The VSLs will have a separate configurable credentials like the BSLs. Like the ObjectStore instantiation, the secret configured in the VSL will be written to disk and passed through the config when calling `Init`. This draft PR for AWS (https://github.com/vmware-tanzu/velero-plugin-for-aws/pull/69) hopefully shows that the two plugins can be configured independently and so the VolumeSnapshotter will not need to rely on the existing default credentials.